packages/xo-web/src/xo-app/jobs/overview/index.js

```diff
@@ -32,12 +29,91 @@ const jobKeyToLabel = {
   genericTask: _('customJob'),
 }
 
+const SCHEDULES_COLUMNS = [
+  {
+    itemRenderer: schedule => (
+      <span>{`${schedule.name} (${schedule.id.slice(4, 8)})`}</span>
+    ),
+    name: _('schedule'),
+    sortCriteria: 'name',
+  },
+  {
+    itemRenderer: (schedule, userData) => {
+      const jobId = schedule.jobId
+      const job = userData.jobs[jobId]
+
+      return (
+        job !== undefined && (
+          <div>
+            <span>{`${job.name} - ${job.method} (${jobId.slice(4, 8)})`}</span>
+            <Link
+              className='btn btn-sm btn-primary ml-1'
+              to={`/jobs/${job.id}/edit`}
+            >
+              <Tooltip content={_('jobEdit')}>
+                <Icon icon='edit' />
+              </Tooltip>
+            </Link>
+          </div>
+        )
+      )
+    },
+    name: _('job'),
+    sortCriteria: (schedule, userData) => {
+      const job = userData.jobs[schedule.jobId]
+      return job !== undefined && job.name
+    },
+  },
+  {
+    itemRenderer: schedule => schedule.cron,
+    name: _('jobScheduling'),
+  },
+  {
+    itemRenderer: schedule => (
+      <StateButton
+        disabledLabel={_('jobStateDisabled')}
+        disabledHandler={enableSchedule}
+        disabledTooltip={_('logIndicationToEnable')}
+        enabledLabel={_('jobStateEnabled')}
+        enabledHandler={disableSchedule}
+        enabledTooltip={_('logIndicationToDisable')}
+        handlerParam={schedule.id}
+        state={schedule.enabled}
+      />
+    ),
+    name: _('jobState'),
+  },
+  {
+    itemRenderer: (schedule, userData) =>
+      userData.isScheduleUserMissing[schedule.id] && (
+        <Tooltip content={_('jobUserNotFound')}>
+          <Icon className='mr-1' icon='error' />
+        </Tooltip>
+      ),
```

Comment:
Does it work without a `name`?
You could put this in the `job` column.

Reply:
it's work without `name`.